src/applications/bmqstoragetool/m_bmqstoragetool_recordprinter.cpp

```diff
@@ -71,7 +71,7 @@ void printRecord(bsl::ostream&                  stream,
     fields.push_back("GUID");
     fields.push_back("Crc32c");
 
-    bmqu::AlignedPrinter printer(stream, &fields);
+    bmqu::AlignedPrinter printer(stream, fields);
```

Comment:
Also need to change type, and do the same in other places in this file:
`bsl::vector<const char*> fields` -> `bsl::vector<bsl::string> fields`